### ISSUE TITLE
GUI Installers! - all these Docker commands and yamls are too error prone during setup!

### ISSUE THREAD

**Issue by guydavis** [guydavis/machinaris#97]
While Docker is a powerful platform, allowing Machinaris to run on all sorts of different systems, the current approach of using `docker run` or `docker-compose.yaml` is really error prone for new users to get right the first time.  This needlessly delays the setup process.

Machinaris needs a cross-platform GUI-installer that will prompt the user for needed information such as:

- Install-mode
- Volume mounts
- Required environment variables

The installer will then either create the Docker compose (or run script), set it to restart automatically, and open the user's browser to the Machinaris site.

**Comment by guydavis** [comment]
For now, I've added docker-compose.yaml and run scripts for Powershell and Bash to the [root folder](https://github.com/guydavis/machinaris).  Let's see how straight-forward this is for new users.

**Comment by jollyjollyjolly** [comment]
I agree the number of issues I have answered on YouTube (bitbybit here) has been clsutered around common-for-docker, yet uncommon-for-layment nomenclature and operational procedures.

**Comment by guydavis** [comment]
Closing as I chose to go with a Docker run/compose Launch Wizard where a user enters their system details into a form and the required/formatted Docker command is output:

1. For the initial install of Machinaris as a fullnode/controller: [https://www.machinaris.app/](https://www.machinaris.app/)
2. Once installed, the Workers page offers a similar Launch Wizard:
![image](https://user-images.githubusercontent.com/12936/129591299-af69c80f-751b-4483-864b-7211ac71866c.png)
Values, such as plotting keys, are automatically filled into the form, greatly minimizing the previous effort to hand-type the correct command.  Hope this helps all new users of Machinaris!
